pkgs/development/python-modules/opencensus-ext-azure/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, azure-core
+, azure-identity
+, opencensus
+, psutil
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "opencensus-ext-azure";
+  version = "1.1.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-UHYIt36djqq2/9X/EbfOuH9U5KapQMS4FDMbTRsDQVE=";
+  };
+
+  propagatedBuildInputs = [
+    azure-core
+    azure-identity
+    opencensus
+    psutil
+    requests
+  ];
+
```

Comment:
Is this running tests?

Reply:
Not really, as it is fetched from PyPi and they haven't included tests into the distribution. The tests are in some (subfolder on github)[https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure] though. But fetching from Github is a bit annoying either as the tags of the actual repository do not match the versions of the extensions in the `contrib/` directory, so maintaining it could be a pita.

Reply:
I'll disable doCheck in the revision